fire/fire_test.py

```diff
@@ -102,6 +103,14 @@ def testFireAnnotatedArgs(self):
     self.assertEqual(fire.Fire(tc.Annotations, 'double 5'), 10)
     self.assertEqual(fire.Fire(tc.Annotations, 'triple 5'), 15)
 
+  @unittest.skipIf(sys.version_info < (3, 0),
```

Comment:
You are correct, and I have fixed it, but these testing libraries tend to only get more version specific over time. You might end up getting back to sys.version_info anyway.